guides/concepts/index.md

```diff
@@ -1 +1,100 @@
-From an outside perspective it is clear that a lot of the underlying concepts in Cardstack reflect some of the useful architectural decisions of Ember. If you are familiar with Ember then you are likely going to understand the most/all of these concepts without too much effort.
+This section is a crash course in Cardstack concepts. It includes vocabulary that you'll find throughout the other topics, and it draws some parallels to other frameworks and JavaScript in general.
+
+
+## Essential Vocabulary
+
+### Cards
+
+Cards are reusable units of functionality - like a blog article, a text editor, a payment system, and more.
+A developer would use many cards, and cards inside of cards, to create the end user experience.
+Each card can have its own data schema, which flows from the front end to the data layer. When someone is looking at a card, it is reflected in the browser URL.
```

Comment:
We should reinforce the fact that a card can contain front end and back end functionality and make it clear that a card is not just a "component"

Reply:
The way I think of a card, is it's like a chunk of an application. Let's take a `post` from a blog application. A `post` card could be the presentation layer of the post, AND the API endpoint that serves the JSON to the front end. A card would handle all of that coordination.

Reply:
Good point, revising this. I like the "vertical slice" framing a lot